openapi-spec/swagger.yaml

```diff
@@ -732,9 +732,9 @@ definitions:
             type: string
   Pool:
     description: >-
-      A backend is initialized by specific driver configuration. Each backend
-      can be regarded as a docking service between SDS controller and storage
-      service.
+      A pool is discoveried and updated by a dock servie. Each pool can be regarded
```

Comment:
it should be changed to ```service```

Reply:
@leonwanghui sorry, what does the service refer to here?

Reply:
I mean there is a spelling error in your patch, just change ```servie``` to ```service```.